packages/react-core/src/components/Button/Button.tsx

```diff
@@ -55,34 +59,66 @@ export const Button: React.FunctionComponent<ButtonProps & OUIAProps> = ({
   isActive = false,
   isBlock = false,
   isDisabled = false,
+  isAriaDisabled = false,
+  inoperableEvents = ['onClick', 'onKeyPress'],
   isInline = false,
   type = ButtonType.button,
   variant = ButtonVariant.primary,
   iconPosition = 'left',
   'aria-label': ariaLabel = null,
   icon = null,
   ouiaId = null,
-  tabIndex = null as number,
+  tabIndex = null,
   ...props
 }: ButtonProps & OUIAProps) => {
   const Component = component as any;
   const isButtonElement = Component === 'button';
+
+  let hasWarnBeta = false;
```

Comment:
Can you remove this var? We can just delete lines 79-85 once it's no longer beta.